air/tests/test_module/features/signatures/algorithms.rs

```diff
@@ -46,18 +50,23 @@ fn test_banned_signature() {
 
     let trace = vec![request_sent_by("init_peer_fake_id")];
 
-    let mut data = serde_json::to_value(InterpreterData::from_execution_result(
+    let mut data_env = InterpreterDataEnv::from_execution_result(
         trace.into(),
         <_>::default(),
         <_>::default(),
         <_>::default(),
         min_supported_version().clone(),
-    ))
-    .unwrap();
+    );
+
+    // let mut data: JValue = InterpreterDataRepr
```

Comment:
obsolete

Reply:
It is not obsolete, it is to make it compile (`#[skip]` is not enough).
Perhaps, it should be excluded completely with some kind of `#[cfg]`.